docs/quickstart.rst

```diff
@@ -263,17 +263,17 @@ Validate webhook signatures
 
     or by using signature method via POST:
 
-    client = nexmo.Client(signature_secret='secret', signature_method='sha256')
+    client = vonage.Client(signature_secret='secret', signature_method='sha256')
 
     if client.check_signature(request.body.decode()):
       # valid signature
     else:
       # invalid signature
 
 Docs:
-`https://docs.nexmo.com/messaging/signing-messages <https://docs.nexmo.com/messaging/signing-messages?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
+`https://docs.vonage.com/messaging/signing-messages <https://docs.vonage.com/messaging/signing-messages?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
```

Comment:
This needs correct to point to the developer docs, this URL isn't valid. `docs.nexmo.com` is also just a redirect, so this should point to the current URL.